.eslintrc.json

```diff
@@ -1,58 +1,65 @@
 {
   "env": {
-    "browser": true,
     "node": true
   },
   "plugins": ["prettier"],
   "extends": ["prettier"],
   "parserOptions": {
     "ecmaVersion": 2020,
-    "sourceType": "module",
+    "sourceType": "script",
     "ecmaFeatures": {
       "jsx": true
     }
   },
   "rules": {
-    "curly": 2,
-    "dot-notation": 2,
-    "id-length": 2,
-    "no-const-assign": 2,
-    "no-dupe-class-members": 2,
-    "no-else-return": 2,
-    "no-inner-declarations": 2,
-    "no-lonely-if": 2,
+    "curly": "error",
```

Comment:
I'm a maintainer of ESLint, and we generally recommend using `"error"` instead of numbers for clarity!